contrib/export/v036/migrate.go

```diff
@@ -0,0 +1,103 @@
+package v036
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	v034gov "github.com/cosmos/cosmos-sdk/contrib/export/v034/gov"
+	v036gov "github.com/cosmos/cosmos-sdk/contrib/export/v036/gov"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/crisis"
+	"github.com/cosmos/cosmos-sdk/x/distribution"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/cosmos/cosmos-sdk/x/mint"
+	"github.com/cosmos/cosmos-sdk/x/slashing"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+)
+
+func migrateGovernance(initialState v034gov.GenesisState) v036gov.GenesisState {
+	var targetGov v036gov.GenesisState
+	targetGov.StartingProposalID = initialState.StartingProposalID
```

Comment:
can we instead do:

```go
targetGov := v036gov.GenesisState{
    DepositParams: initialState.DepositParams,
    // ...
}
```

Reply:
It's nicer 👍  I wasn't sure because for some elements I needed to iterate, worst case I'll start them empty and fill them later on